app/src/components/integrations/modals/addIntegrationModal/addIntegrationModal.jsx

```diff
@@ -62,13 +63,25 @@ const AddIntegrationModal = ({ data, initialize, change, handleSubmit }) => {
   const fieldsExtensions = useSelector(uiExtensionIntegrationFormFieldsSelector);
   const dispatch = useDispatch();
   const { formatMessage } = useIntl();
-  const { onConfirm, customProps, isGlobal } = data;
+  const { trackEvent } = useTracking();
+  const {
+    onConfirm,
+    customProps,
+    isGlobal,
+    eventsInfo: { createGlobalIntegration },
+  } = data;
+
+  const integrationFieldsExtension = fieldsExtensions.find(
+    (ext) => ext.pluginName === data.instanceType,
+  );
 
   const updateMetaData = (newMetaData) => {
     setMetaData({ ...metaData, ...newMetaData });
   };
 
   const onSubmit = (newData) => {
+    trackEvent(createGlobalIntegration(integrationFieldsExtension.pluginName));
```

Comment:
This modal used to create project integrations as well.
So please use more generic name here.
Also can we avoid providing events from props in that case? As this modal used several times for different plugins and default props won't work for most of the cases.
It supports aka 4 mods: create global + editAuth for global, create project + editAuth for project.
You can determine the right event using the existing props `isGlobal` & `editAuthMode`.